docs/book/getting-started/core-concepts.md

```diff
@@ -28,7 +28,7 @@ Artifacts represent the data that goes through your steps as inputs and outputs
 
 #### Materializers
 
-Materializers define how Artifacts live in-between steps. More precisely, they define how data of a particular type can be serialized/deserialized, so that the steps are able to load the input data and store the output data.
+Materializers define how artifacts live in-between steps. More precisely, they define how data of a particular type can be serialized/deserialized, so that the steps are able to load the input data and store the output data.
```

Comment:
```suggestion
Materializers define how artifacts live in between steps. More precisely, they define how data of a particular type can be serialized/deserialized, so that the steps are able to load the input data and store the output data.
```